doc/csv/recipes/generating.rdoc

```diff
@@ -111,3 +114,36 @@ Use class method CSV.new without option +headers+ to generate \CSV data to an \I
     csv << ['Baz', 2]
   end
   p File.read(path) # => "Foo,0\nBar,1\nBaz,2\n"
+
+=== Converting Fields
+
+You can use _write_ _converters_ to convert fields when generating \CSV.
+
+==== Recipe: Filter Generated Field Strings
+
+Use option <tt>:write_converters</tt> and a custom converter to convert field values when generating \CSV.
```

Comment:
We can use `+...+` for `<tt>...</tt>` in RDoc.

Reply:
Sorry, @kou. I thought (wrongly, it turns out) that the colon meant that <tt> is needed.  There are a couple of dozen of these in this file, and dozens more in other files.  If you want them changed, I suggest doing all at once in a separate PR.

Reply:
All fixed in the one file.